types/auto/modules/sdp/SDP.d.ts

```diff
@@ -29,6 +29,12 @@ export default class SDP {
      * @type {boolean}
      */
     removeUdpCandidates: boolean;
+    /**
+     * Adds a new m-line for receiving source of the given media type.
+     *
+     * @param {Mediatype} mediaType media type of the new source that is being added.
+     */
+    addMlineForNewLocalSource(mediaType: any): void;
```

Comment:
is there no type definition for `mediaType`?

Reply:
These are auto generated types based on the SDP JS file. This is the inferred type since type is not specified in the JS file.